src/components/Footer/Footer.tsx

```diff
@@ -22,7 +22,7 @@ const Footer: React.FC = () => {
   return (
     <Box className={classes.footer}>
       <QuickIcon />
-      <Typography>© 2021 QuickSwap.</Typography>
+      <Typography>© 2022 QuickSwap.</Typography>
```

Comment:
Did we actually get this copy written? Or is this just superficial to look more official.

If this is suppose to update yearly you should probably grab the year from the days date.

Reply:
updated copyrightYear using `const copyrightYear = new Date().getFullYear();` at the moment